### ISSUE TITLE
ARES - Various shaders do not work as expected

### ISSUE THREAD

**Issue by Yotamtamtam** [higan-emu/ares#294]
Tested on the nightly build of Ares - 11.11.2021
Tested on Windows 10 Home. V. 10.0.19042, Intel i7-7700HQ @ 2.8GHz, Geforce GTX 1050 Ti

Issues:
1) PAL-r57shell shader crash the emulator and perform exit operation.
2) Phosphorish shader seems to render different graphics according the gui window size on the Mega-Drive core.
See attached images.

Small Window (ares deafult window size):
![shader_p_window](https://user-images.githubusercontent.com/87193210/141608649-d5d7ba53-562b-49f7-a023-c5bc6dc59a6c.png)

Bigger window (stretch a little bit the window size):
![shader_p_bigger_window](https://user-images.githubusercontent.com/87193210/141608646-d16712b6-cc80-4adb-ba46-32e0032e3568.png)

Full Screen:
![shader_p_fscreen](https://user-images.githubusercontent.com/87193210/141608648-874b0fb2-ba8b-4ec5-9592-b4ea8b109024.png)

**Comment by kyjexthu** [comment]
The shaders seem to be not scaling properly with the output resolution from what I can tell. With the scanlines shader for example, there are far too many scanlines and they don't line up with the pixels when scaled to higher resolutions.

**Comment by mahougomi** [comment]
I experience the same issue with the scanlines shader, but the smallest possible window size is 2x, meaning that the scanlines _never_ line up correctly.

**Comment by Yotamtamtam** [comment]
Still occur on the nightly build of Ares - 7.02.2022

**Comment by Yotamtamtam** [comment]
Moreover, Ive noticed that the Super-Nintendo graphics are a bit unclear in it's deafult state (with no shaders).
See attached photos for comparison - ares and bsnes.

![E02](https://user-images.githubusercontent.com/87193210/153700112-65df8961-334a-4de0-ada9-7736525fd9b2.png)
![E01](https://user-images.githubusercontent.com/87193210/153700118-1bd5fae4-9a59-4465-8aff-bedd93f0c1ea.png)

**Comment by Sunspark-007** [comment]
Still an issue on 130.1

**Comment by Kugelblitz360** [comment]
pal-r57shell still crashes Ares V132 (exit to desktop) when selected.

**Comment by remutro** [comment]
@Yotamtamtam  -  This issue was opened over 3 years ago, long before librashader was introduced. I just checked r57shell and it doesn't cause any crashes at all. And phosphorish doesn't seem to appear in the new shader set at all. I have a feeling we don't need to keep this issue open anymore - can you please comment (or just close if no longer valid)?

**Comment by Kugelblitz360** [comment]
Yeah, checked a couple of shaders on v141 and all worked, so I'd say close this issue.

**Comment by Yotamtamtam** [comment]
Absolutely. The issue was opened and was relevant prior to the implementation of Librashader.
Thank you for your attention. I'm closing this thread.
